samples/hip_sycl_interop/hip_sycl_interop.cpp

```diff
@@ -61,7 +61,13 @@ void VerifyResult(float *c_A, float *c_B) {
 }
 
 int main() {
-  std::string envVar = std::getenv("CHIP_BE");
+  const char* val = std::getenv("CHIP_BE");
+  // In case CHIP_BE is not present
+  if (!val) {
+    std::cout << "HIP_SKIP_THIS_TEST" << std::endl;
+    exit(0);
+  }
+  std::string envVar(val);
```

Comment:
the runtime will default to something for all other tests so we should query the runtime to get this value rather than reading the env.